topbar.js

```diff
@@ -188,10 +245,72 @@ class WorkspaceMenu extends PanelMenu.Button {
             }
         });
 
-        this.menu.box.add(this.prefsIcon, { expand: true, x_fill: false });
+        function rotated(list, dir=1) {
+            return [].concat(
+                list.slice(dir),
+                list.slice(0, dir)
+            );
+        }
+
+        let cycle = (dir=1) => {
```

Comment:
Move to a util to facilitate usage from keybindings?

Reply:
Yes, probably settings.js, not entirely sure where it fits in the best.

Reply:
Move to tiling.